lib/internal/errors.js

```diff
@@ -1207,7 +1207,8 @@ E('ERR_UNHANDLED_ERROR',
 E('ERR_UNKNOWN_BUILTIN_MODULE', 'No such built-in module: %s', Error);
 E('ERR_UNKNOWN_CREDENTIAL', '%s identifier does not exist: %s', Error);
 E('ERR_UNKNOWN_ENCODING', 'Unknown encoding: %s', TypeError);
-E('ERR_UNKNOWN_FILE_EXTENSION', 'Unknown file extension: %s', TypeError);
+E('ERR_UNKNOWN_FILE_EXTENSION', 'Unknown file extension %s imported from %s',
```

Comment:
Perhaps we should explicitly add the name of the extension here? `Unknown file extension %s for %s imported from %s`?

Reply:
Yeah that's clearer. Added the changes